docs/broadcasters/getting-started/playback-livestream.md

```diff
@@ -5,6 +5,12 @@ sidebar_position: 5
 
 ## Playback a Livestream
 
+### Using the Aqueduct Dashboard
```

Comment:
Would it make sense to add code examples that use the MistServer video player?  It supports lots of different playback protocols, could be a nice value add for users.

Reply:
I think so, yeah. Do you know where I could find a sample implementation? otherwise I can ask someone on the Leiden team to provide one